packages/server/lib/server.ts

```diff
@@ -22,9 +22,18 @@ import { refreshConnectionsCron } from './refreshConnections.js';
 const { NANGO_MIGRATE_AT_START = 'true' } = process.env;
 const logger = getLogger('Server');
 
+process.on('unhandledRejection', (reason) => {
+    logger.error('Received unhandledRejection...', reason);
+    // not closing on purpose
+});
+
+process.on('uncaughtException', (e) => {
+    logger.error('Received uncaughtException...', e);
+    // not closing on purpose
+});
+
 const app = express();
 app.disable('x-powered-by');
-app.set('trust proxy', 1);
```

Comment:
Are you sure about that, it's usually mandatory as you as you have a load balancer

Reply:
Gah, there's something crazy that keeps happening with eslint where it prunes lines randomly when I save.